blog/2021-07-15-supervisor_update.md

```diff
@@ -0,0 +1,72 @@
+
+The old [`/snapshots`][supervisor_api_snapshots] endpoints are now deprecated and are scheduled for removal in Q4 of this year.
+
+### Backup structure changes
+
+For consistency, we have also changed the name of the meta file inside the backup tar from `snapshot.json` to `backup.json`, so if you have a tool that use that file, you need to adjust it to look for both.
+
+## Streaming ingress
+
+Some add-ons need to receive large payloads from the user, for instance with uploading. Previously, there has been a limit of 16 MB per request for add-ons running behind ingress and this is still the default. If you need to receive bigger payloads, you can enable this by setting `ingress_stream` to `True` in the add-on configuration. When you do this the request is streamed from the client to your add-on, which has no size limit and virtually no overhead.
+
+Note that not all webservers are able to handle this by default, so you might need to adjust it.
+
+## Deprecated API endpoints
+
+Over the past years we have restructured parts of our API endpoints, but we have also kept old endpoints working. If you use any of the deprecated endpoints in your tools/add-ons you should adjust this. All deprecated endpoints are scheduled for removal in Q4 this year.
+
+Here is a list of those and their replacements:
+
+Deprecated endpoints | Replaced with
+-- | --
+`/homeassistant/*` | `/core/*`
+`/snapshots/*` | `/backups/*`
+
+In addition to this, the following are also deprecated and are also scheduled for removal in Q4 this year.
+
+- The environment variable `HASSIO_TOKEN` has been replaced with `SUPERVISOR_TOKEN`.
+- Using `X-Hassio-Key` header has been replaced with [using `Authorization` with a Bearer token][api_example].
+- Using `http://hassio/` to communicate with the Supervisor has been replaced with `http://supervisor/`.
+
+## Supervised installation
+
+Maintaining a [supervised installation][supervised_installation] is currently not the best experience. [The script][supervised_script] that most users use to install is behind what the Supervisor wants from the host. Since there are no real upgrade paths for those using it, users of it need to manually adjust their installation.
+
+Recently we created the [OS Agent][os_agent] as mentioned in the [blog on the main site][main_blog]. This allows for better communication between the host OS and the Supervisor, and to bring in more features. All current supervised installations now have to install this manually to take advantage of it.
+
+An alternative to this route is to package and distribute it as a deb package that can be installed and upgraded with `apt` on the host. For this to be viable, we are looking for a person (or a group of people) that wants to create and maintain this type of deployment, and bring the supervised installation method up to par with our OS, and more importantly make updates needed on the host easier for the users.
```

Comment:
Is it the Supervisor we want to package?
```suggestion
An alternative to this route is to package and distribute the Supervisor as a deb package that can be installed and upgraded with `apt` on the host. For this to be viable, we are looking for a person (or a group of people) that wants to create and maintain this type of deployment, and bring the supervised installation method up to par with our OS, and more importantly make updates needed on the host easier for the users.
```

Reply:
Everything needed for a supervised installation

Reply:
Right, then I think my suggestion is ok.

Reply:
I think that is misleading, `distribute the supervised installation as a deb package` maybe?

Reply:
Sound good!

Reply:
```suggestion
An alternative to this route is to package and distribute the supervised installation as a deb package that can be installed and upgraded with `apt` on the host. For this to be viable, we are looking for a person (or a group of people) that wants to create and maintain this type of deployment, and bring the supervised installation method up to par with our OS, and more importantly make updates needed on the host easier for the users.
```